src/backend/bacon-video-widget-gst-missing-plugins.c

```diff
@@ -521,30 +585,31 @@ show_codec_missing_information_dialog (TotemCodecInstallContext *ctx)
 	                                 GTK_MESSAGE_ERROR,
 	                                 GTK_BUTTONS_CANCEL,
 	                                 _("Unable to play the file"));
+	action = determine_vlc_launch_action_for_missing_codecs (bacon_video_widget_get_mrl (ctx->bvw));
 
-	/* Show additional button pointing to our online store if the
-	 * missing codec can be installed via our codecs activation key. */
-	if (missing_codecs_supported ((const char**)ctx->details)) {
```

Comment:
All that logic is actually introduced in other downstream patches we ship; I don't think there's any benefit to keeping it here, unless we wanted an option to choose whether to display the old popup or this one.
Since I don't think we do for now, and we can always bring that back at a later stage, please revert the commits that introduce the old codec popup.

Alternatively, we should probably make it a gsetting (but I don't think that'd be useful right now)